views/index.ejs

```diff
@@ -34,18 +34,17 @@
         let outputText = ""
 
         splitInput = words.match(/\w+|"[^"]+"/g);
-        console.log(splitInput)
+
         splitInput.forEach((x, y) => {
-          console.log(x)
-          console.log(y)
           splitInput[y] = x.split('"').join('')
         });
 
         splitInput.forEach((x) => {
-          outputText = sourceText.split(x).join('XXXX')
+          sourceText = sourceText.split(x).join('XXXX')
```

Comment:
The cause of the issue before was that I was setting this `sourceText` variable to an accumulator previously defined above. This was re-setting it every time it went through the loop.